src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -26967,5 +26967,109 @@ public class Outer
             CompileAndVerify(compRelease, expectedOutput: "C1 { I1 = 42 }", verify: Verification.Skipped /* init-only */);
             compRelease.VerifyEmitDiagnostics();
         }
+
+        [Fact]
+        [WorkItem(44571, "https://github.com/dotnet/roslyn/issues/44571")]
+        public void XmlDoc()
+        {
+            var src = @"
+/// <summary>Summary</summary>
+/// <param name=""I1"">Description for I1</param>
+public record C(int I1);
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>Ignored</summary>
+    public static class IsExternalInit
+    {
+    }
+}
+";
+
+            var comp = CreateCompilation(src, parseOptions: TestOptions.RegularWithDocumentationComments);
+
+            var tree = comp.SyntaxTrees[0];
+            var model = comp.GetSemanticModel(tree, ignoreAccessibility: false);
+            var parameter = tree.GetRoot().DescendantNodes().OfType<ParameterSyntax>().First();
+            var i1 = model.GetDeclaredSymbol(parameter);
+
+            comp.VerifyDiagnostics(
+                // (4,21): warning CS1591: Missing XML comment for publicly visible type or member 'C.I1'
+                // public record C(int I1);
+                Diagnostic(ErrorCode.WRN_MissingXMLComment, "I1").WithArguments("C.I1").WithLocation(4, 21)
+                );
+
+            var cMember = comp.GetMember<NamedTypeSymbol>("C");
+            Assert.Equal(
+@"<member name=""T:C"">
+    <summary>Summary</summary>
+    <param name=""I1"">Description for I1</param>
```

Comment:
><param name=""I1"">Description for I1</param> [](start = 4, length = 45)

Is it Ok to have a param tag on the type?

Reply:
Yes, this is what we had discussed. See notes: https://github.com/dotnet/roslyn/issues/44571#issuecomment-659448287